packages/test-utils/src/wrapper.js

```diff
@@ -99,8 +100,7 @@ export default class Wrapper implements BaseWrapper {
   contains (selector: Selector) {
     const selectorType = getSelectorTypeOrThrow(selector, 'contains')
     const nodes = findAll(this.vm, this.vnode, this.element, selector)
-    const is = selectorType === REF_SELECTOR ? false : this.is(selector)
-    return nodes.length > 0 || is
+    return nodes.length > 0 || (selectorType !== REF_SELECTOR && this.is(selector))
```

Comment:
Why is method cannot just return `false` for REF_SELECTOR instead of throwing error ?